src/scss/skin-super-modern/components/_seekbar.scss

```diff
@@ -46,15 +46,19 @@ $seekbar-height: .3125em;
     @include hidden;
     @include focusable;
 
-    border-radius: 1em;
     cursor: pointer;
     font-size: 1em;
     height: $seekbar-height;
     margin: calc((1em - $seekbar-height) / 2) 0;
-    overflow: hidden;
```

Comment:
Having the overflow hidden here breaks the playback position marker as it is bigger than this container.